index.html

```diff
@@ -3650,14 +3650,115 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 
   <p id="overriding-protected-terms">
     While protected terms can in general not be overridden,
-    there is an exception to this rule:
-    a property-<a>scoped context</a> is not affected by protection,
-    and can therefore override protected terms,
+    there are two exceptions to this rule.
+    The first exception is that a context is allowed to redefine a protected term
+    if the new definition is identical to the protected term definition
+    (modulo the <code>@protected</code> flag).
+    The rationale is that the new definition does not violate the protection,
+    as it does not change the semantics of the protected term.
+    This is useful for widespread term definitions,
+    such as e.g. aliasing <code>@type</code> to <code>type</code>,
```

Comment:
"such as e.g." seems repetitive.  How about:
```suggestion
    such as aliasing <code>@type</code> to <code>type</code>,
```